etstool.py

```diff
@@ -227,6 +224,16 @@ def install(edm, runtime, environment, editable, docs, source):
     click.echo("Done install")
 
 
+def _get_install_command_string(pkg_or_location, editable, no_deps=True):
```

Comment:
Please could you add a docstring to this function, and move it down with the rest of the utility functions so that we keep the clean organization of this file (click commands all together at the top, utility functions further down).